### DISCUSSION TITLE
Creating a new note

### DISCUSSION THREAD

**Discussion by eamers** [glowingjade/obsidian-smart-composer#169] [Q&A] [answered]
Currently when Smart Compose suggests creating new note (and suggests the content for this note) I can click "Apply", but it always applies the suggestion to whatever note I currently have open. In cursor I have the ability to create the suggest file and then apply the suggestions. Am I missing something or does this functionality just not exist yet?

BTW This plugin is incredible, I feel so much gratitude for how is already helping me

**Comment by glowingjade** [comment]
Will fix this soon. Thanks for your suggestion!

**Comment by glowingjade** [comment, marked as answer]
I've created a dedicated issue to track this feature request: https://github.com/glowingjade/obsidian-smart-composer/issues/247. Closing this discussion to consolidate feedback in the issue.
